jose/jws.py

```diff
@@ -98,7 +98,7 @@ def get_unverified_headers(token):
     compatibility.
 
     Args:
-        token (str): A signed JWS to decode the headers from.
+        token (str): A signed JWS to decode the claiams from.
```

Comment:
You misspelled "claims" and also this is in the `get_unverified_headers` not `get_unverified_claims`

Reply:
Ironic, I made the kind of mistake in pr that I created this pr to fix.